### ISSUE TITLE
Maven artifact is referencing itself

### ISSUE THREAD

**Issue by wilx** [javafxports/openjdk-jfx#330]
I am using version 11.0.1 and I get the following error when I try to run `mvn site:site`:

```
[WARNING] Unable to create Maven project from repository.
org.apache.maven.project.ProjectBuildingException: 1 problem was encountered while building the effective model for org.openjfx:javafx-graphics:11.0.1
[FATAL] 'dependencies.dependency org.openjfx:javafx-graphics:11.0.1' for org.openjfx:javafx-graphics:11.0.1 is referencing itself. @ line 13, column 17
 for project org.openjfx:javafx-graphics:11.0.1
    at org.apache.maven.project.DefaultProjectBuilder.build (DefaultProjectBuilder.java:165)
```

**Comment by wilx** [comment]
Cross reference: [[MNG-6541] less restrictive "is referencing itself" check - ASF JIRA](https://issues.apache.org/jira/browse/MNG-6541?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel&focusedCommentId=16727069#comment-16727069)

**Comment by brcolow** [comment]
Out of curiosity, what does running `mvn site:site` on this repository do and is it useful? What reports would it generate?

**Comment by wilx** [comment]
[**@brcolow**](https://github.com/brcolow) commented on [Jan 3, 2019, 4:00 PM GMT+1](https://github.com/javafxports/openjdk-jfx/issues/330#issuecomment-451168212 "2019-01-03T15:00:15Z - Replied by Github Reply Comments"):
> Out of curiosity, what does running `mvn site:site` on this repository do and is it useful? What reports would it generate?

This bug report is not about running `mvn site:site` in clone of this repository. It is about running `mvn site:site` in a project that uses JavaFX artifacts provided by this repository.

**Comment by brcolow** [comment]
Got it, sorry about that.

**Comment by wilx** [comment]
Apparently, this issue was fixed in Maven 3.6.0.
